src/AnkiService.ts

```diff
@@ -21,8 +21,10 @@ export class AnkiService {
   }
 
   async invoke(action: string, params?: object): Promise<any> {
+    console.log("{ ...params }: %s", { ...params });
```

Comment:
I'm assuming you're still experimenting here and these would eventually be removed or changed to logger

Reply:
Oversight on my part